cmd/e2etest/e2e_test.go

```diff
@@ -1,4 +1,4 @@
-package test
+package e2etest
```

Comment:
maybe we can use e2e_test name. for go pkg name is "wordword", but cmd can be "word_word". maybe e2e_test is a main process would be better